<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandingPage</class>
 <widget class="QMainWindow" name="LandingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>433</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 161, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(104, 161, 229);</string>
    </property>
    <property name="text">
     <string>My health account </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 } 


</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(242, 248, 253,0)</string>
     </property>
     <property name="text">
      <string>Login or Signup </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSignup">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>ToolTip</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
	border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
	border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
	border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
	border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(242, 248, 253,0)</string>
     </property>
     <property name="text">
      <string>or</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 } 


</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <bold>true</bold>
         <underline>true</underline>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>What I can do?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>View:                  RAT test results
View:                  Vaccination record
Request:            Vaccine Pass
Report:               RAT test result
Report:               Issues</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>341</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelLink">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(104, 161, 229, 0);</string>
     </property>
     <property name="text">
      <string>External Link</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
